core/build.gradle

```diff
@@ -287,6 +290,7 @@ dependencies {
   implementation project(':auth-connector')
   implementation project(':secret-engine')
   implementation project(':template-manager-core')
+  implementation project(':autoscale')
```

Comment:
No, it is a very bad idea to add dependencies between microservice modules. You shall rely on `autoscale-api` that contains the respective client and endpoint types.